src/main/java/com/minelittlepony/hdskins/gui/GuiSkinsMineLP.java

```diff
@@ -22,26 +26,26 @@ protected EntityPlayerModel getModel(GameProfile profile) {
     }
 
     @Override
-    protected void onSetLocalSkin(MinecraftProfileTexture.Type type) {
+    protected void onSetLocalSkin(Type type) {
         MineLittlePony.logger.debug("Invalidating old local skin, checking updated local skin");
-        if (type == MinecraftProfileTexture.Type.SKIN) {
```

Comment:
I would prefer you keep the Owner class. It makes it easier to tell what `Type` is.

Reply:
You don't need to know the name of the owner class. It's superfluous information you can pick from the import statement. Just like `this`, all your doing is increasing the amount of text one has to read to understand what the code is doing.